HelloEndpoints/README.md

```diff
@@ -134,6 +134,7 @@ If your backend is working locally, you can deploy it to Google App Engine.
 sign-in with your Google Account. Choose an account and sign in.<br>
 
 4. Create a new project and switch back to the **Deploy to App Engine** dialog in Android Studio.
+    + You may also need to initialize the App Engine before being able to deploy a new project. To do so, go to the [Google Developers Console](https://console.developers.google.com) and run the following command: `gcloud beta app create`
```

Comment:
BTW I think you meant cloud shell?  It may actually be better just to send them to "https://console.cloud.google.com/appengine", which takes the user through App Engine intialization and region selection.

Thanks again.